src/services/WebSocketPolyfill.js

```diff
@@ -88,19 +88,33 @@ export default function initWebSocketPolyfill(syncService, fileId, initialSessio
 		}
 
 		#initiateSending() {
-			let steps
+			let outbox
 			syncService.sendSteps(() => {
-				steps = this.#queue.map(s => encodeArrayBuffer(s))
+				outbox = this.#queue
```

Comment:
When testing it seems that debouncing needs to be improved now for the push request, it seems on my test with two browsers the push is triggering every 200ms through the following stack trace even if the cursor positions do not change:

<img width="1213" alt="Screenshot 2023-02-08 at 12 10 59" src="https://user-images.githubusercontent.com/3404133/217513863-136b06b8-7e5e-4b3a-ac8b-9508003543d1.png">

Reply:
I suspect this is due to us replaying old awareness messages and creating an inconstent state.
I created #3785 to track it.